ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -183,6 +183,30 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# Route Filtering #}
```

Comment:
Let us make a common OSPF Areas documentation, so we only need to check if `router_ospf.process_ids[process].areas` is defined. We will add more columns in future PRs